lib/language.js

```diff
@@ -73,7 +73,8 @@ exports.errors = {
         arity: 'must have an arity of {{n}}',
         minArity: 'must have an arity greater or equal to {{n}}',
         maxArity: 'must have an arity lesser or equal to {{n}}',
-        ref: 'must be a Joi reference'
+        ref: 'must be a Joi reference',
+        class: 'must be an ES6 class'
```

Comment:
Wouldn't we just say "must be a class"? We shouldn't be having an `ES6 Class` and then later an `ES7 Class`. (This is a bit of a pedantic comment, but I'd rather make it than not)